pyproject.toml

```diff
@@ -53,12 +53,12 @@ n2n = "nyaml.cli:launch_tool"
 
 [tool.ruff]
 include = ["pynxtools/*.py", "tests/*.py"]
```

Comment:
Could you also change this `pynxtools` occurrences and replace it with nyaml. It seems that I forgot to change this when I created the repo

Reply:
Yes, it looks all good now. Thanks!